.github/workflows/lint-js.yml

```diff
@@ -1,6 +1,8 @@
 name: JavaScript Linting
 on:
   push:
+    branches-ignore:
+      - "dependabot/**"
```

Comment:
Oh now I see why you are skipping this in PR #1523, approving there and here!

Reply:
Yup, just avoids those times when an unstable test would pass on the PR and not the branch or vice-versa :)